### DISCUSSION TITLE
[Question] Why is fix_base defaulted to MISSING, not True?

### DISCUSSION THREAD

**Discussion by hyoungmin94** [isaac-sim/IsaacLab#1339] [Q&A] [answered]
https://isaac-sim.github.io/IsaacLab/main/_modules/omni/isaac/lab/sim/converters/urdf_converter_cfg.html#UrdfConverterCfg

I was looking at this code, only to figure out that  fix_base is set as MISSING as default, even when the dsecription says it's defaulted to True. Is this a mistake, or am I not understanding something quite correctly?

**Comment by StrainFlow** [comment, marked as answer]
Thank you for catching that! I've spoken with the engineers and MISSING is the expected default value, so the documentation comment is incorrect. I've filed a bug and that should be fixed in the next release.
